src/ape_compile/__init__.py

```diff
@@ -23,6 +26,56 @@ class Config(PluginConfig):
     Source exclusion globs across all file types.
     """
 
+    cache_folder: Optional[Path] = None
+    """
+    Path to contract dependency cache directory (e.g. `contracts/.cache`)
+    """
+
+    @property
+    def base_dir(self) -> Path:
```

Comment:
suggestion: rename to `base_path` to be consistent with the compiler API parameter names